ckanext/unaids/tests/test_plugin.py

```diff
@@ -30,3 +33,61 @@ def test_geojson_format_guessed_correctly(self):
         response = call_action('resource_create', {}, **resource)
         response = call_action('package_show', {}, id=dataset['id'])
         assert response['resources'][0]['format'] == 'GeoJSON'
+
+
+class TestResourceLastModified(object):
+    '''Tests for the ckanext.plugin._update_resource_last_modified_date module.
+
+    Make sure we are setting the last_modified to the resource dict
+    whenever a resource is created or edited.
+    '''
+
+    URL_TYPE_FILE_UPLOAD = 'upload'
+    URL_TYPE_LINK = ''
+
+    def test_resource(self, url_type):
```

Comment:
this method breaks the tests. Prefix `test_` in a method name makes pytest to pick this up as a test and adds it to the suite.
https://docs.pytest.org/en/6.2.x/goodpractices.html#conventions-for-python-test-discovery